src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let newObj = { ...state };
+  const resArr = [];
```

Comment:
Let's create more meaningful variable names and don't use names of data types in them

Reply:
> Let's create more meaningful variable names and don't use names of data types in them

understood